### PR TITLE
Update improve-modules.md

### PR DESCRIPTION
**Pull request by clarkenj**
Since we will now be creating maintenance issues, I thought this template might be more useful for us?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/improve-modules.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 about: Maintain the content of existing module.
 title: ''
 labels: Enhance
 assignees: ''
 ---
 
-## Which module you would like to maintain?
-<!--have a look at the listed modules https://school-brainhack.github.io/modules/ -->
-<!--Tell us which module you want to maintain-->
+<!--Title = "Maintentance of ****" e.g. Maintenance of DL_for_neuroimaging -->
```
Comment (htwangtw) [inline .github/ISSUE_TEMPLATE/improve-modules.md:9]: You can put this template in line 4

**Review by htwangtw — APPROVED** [review]
Some minor things otherwise LGTM

**Comment by clarkenj** [comment]
Thanks for the suggestions, done

File: .github/ISSUE_TEMPLATE/improve-modules.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ---
 name: Maintenance of module
 about: Maintain the content of existing module.
-title: ''
+title: 'e.g. Maintenance of DL_for_neuroimaging'
```
Comment (htwangtw) [inline .github/ISSUE_TEMPLATE/improve-modules.md]:
```suggestion
title: 'Maintenance of [module name]'
```

**Pull request merged**